ElectronicsStore/ElectricalDevice.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ElectronicsStore
+{
+    public abstract class ElectricalDevice : ISwitch
+    {
+
+        protected string? ModelName;
+        protected decimal Price;
+
+        public ElectricalDevice (string? modelName, decimal price)
+        {
+            ModelName = modelName;
+            Price = price;
+        }
+
+        public abstract string Description { get; }
+    
+
+        public void TurnOn()
+        {
+            Console.WriteLine("Press button at the top");
+        }
+
+        public void TunrnOff()
```

Comment:
All inherited classes have different implementation of `TurnOff()` method.
And please fix typo))

Reply:
I don't know it looks like TurnOff() method is the same in all classes